code/game/objects/items/storage/pouch.dm

```diff
@@ -320,6 +320,24 @@
 		/obj/item/ammo_magazine/standard_smartmachinegun,
 	)
 
+/obj/item/storage/pouch/explosive/engineerdetpack/Initialize()
```

Comment:
XD This is so bad.

Ok so, when you make a proc for a typepath that isnt defined, it creates the typepath regardless. It just inherits from the parent.

So basically you dont want all these definitions inside the Initialize() for goodness sakes.

Reply:
Yeah, I'm not going to bother making a new pouch.